src/server/models/cacheable_queries/user.js

```diff
@@ -97,7 +99,11 @@ const loadUserRoles = async userId => {
       const userRoles = {};
       Object.keys(roles).forEach(orgId => {
         const [highestRole, orgName] = roles[orgId].split(":");
-        userRoles[orgId] = { id: orgId, name: orgName, role: highestRole };
+        userRoles[orgId] = {
+          id: Number(orgId),
```

Comment:
while writing tests, this would be a string or a number depending on whether it came from the db or cache, so I'm making it explicit here to be consistent